core/src/test/scala/integration/kafka/api/LogDirFailureTest.scala

```diff
@@ -64,6 +65,35 @@ class LogDirFailureTest extends IntegrationTestHarness {
   }
 
   @Test
+  def brokerWithOldIBPShouldHaltOnLogDirFailure() {
```

Comment:
Nit: the Kafka conversion is to capitalise the first letter only. So, `IBP` should be `Ibp`. But perhaps we should simply write it out, i.e. `brokerWithOldInterBrokerProtocolShouldHaltOnLogDirFailure`. Also, we can maybe specify with version is old in the method doc.

Reply:
Thanks for the suggestion. I will keep this convention in mind going forward.

I will update the patch to use the suggested name and specify which version is old in the method doc.